src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -97,6 +106,17 @@ public void handleObservers(JButton button) {
         this.isDeleted = !this.isDeleted;
     }
 
+    private void loadCommandsFromJSON() {
+        String jsonInput = InputCommandsTextArea.getText().trim();
+        JSONCommandParser jsonCommandParser = new JSONCommandParser(jsonInput);
```

Comment:
You should be able to use parser using only Parser interface! If not, this interface has no value.